new_words_spell_checker.txt

```diff
@@ -57,3 +57,4 @@ Soltan
 systemd
 Tosta
 Zeestraten
+Heitor
```

Comment:
I wonder why this is needed? Can you keep this file alphabetically ordered please?

Reply:
Without it, the spell checker fails. I am gonna get it ordered.